### DISCUSSION TITLE
Yolov8 for Semantic Segmentation

### DISCUSSION THREAD

**Discussion by qasim7520** [ultralytics/ultralytics#4834] [Q&A]
can we use yolov8 or v5 for semantic segmentation ?

**Comment by glenn-jocher** [comment]
@Qasim7520 yes, you can use YOLOv8 for semantic segmentation tasks. YOLOv8 includes the Segment Anything Model (SAM), which is designed for segmentation purposes. While YOLOv5 does not natively support semantic segmentation, YOLOv8's SAM can be a powerful tool for this task. For more information on how to use YOLOv8 for segmentation, please refer to the segmentation section in our documentation. Happy segmenting! 😊🎨
